### PR TITLE
changed for ansible tower

### PR DESCRIPTION
**Pull request by shay6**
turn postgresql_user to root

design README

add python-psycopg2 to required packages

install firewalld so systemd module will go smoothly

move requirements.yml to roles folder so ansible tower will download required roles

remove loop to prevent ansible tower bug: https://bugzilla.redhat.com/show_bug.cgi?id=1600124

remove roles folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+ -  Inventory
+
+    - Create inventory of 2 hosts. The name of the hosts would be "db" and "ui".
+    - Include the next extra_vars for every host:
+    - ansible_host: {Enter your host here}
+    - ansible_ssh_user: root 
+
+ - Project
+
+    - Create project for sonar and connect it to our git repo
```
Comment (abraverm) [inline README.md]: s/our//

File: README.md [hunk 2]
```diff
@@ -1,3 +1,32 @@
-Install required roles by:
+### Install required roles by:
```
Comment (abraverm) [inline README.md:1]: s/our//g

File: playbook.yml [hunk 3]
```diff
@@ -22,8 +22,9 @@
     when: sonar == "db"
 
   - include_role:
-      name: "{{ item }}"
+      name: geerlingguy.java
```
Comment (abraverm) [inline playbook.yml]: add a comment with link to the bug you found

Comment (shay6) [reply playbook.yml]: to add the link in the playbook itself?

File: README.md (hunk 2 above)
Comment (shay6) [reply README.md:1]: you could write:
sed -i 's/our//g' README.md

File: playbook.yml (hunk 3 above)
Comment (abraverm) [reply playbook.yml]: yes

File: README.md (hunk 2 above)
Comment (shay6) [reply README.md:1]: done

**Pull request merged**